index.js

```diff
@@ -847,6 +955,303 @@ function flip(message, side, coinage) {
+            return;
+        }
+        //check that the user is not also selling
+        if (id in market[1]){
+            response += ', you can\'t create a buy posting, you are already selling. ya dingus!\n';
+            message.channel.send(response);
+            message.author.send('Your posting:\n' + id.padStart(18,'0') + '\tS ' + String(market[1][id].quantity).padStart(10,' ') + '\t@' + String(market[1][id].price).padStart(8,' ') + '\n');
+            return;
+        }
+        response += ', is buyin\' who\'s sellin\'?';    
+
+        // ledger
+        userData.gold -= value;
+        var userData = ledger[id];
+        market[0][id] = {
+                username: ledger[id].username,
+                price: parseFloat(price),
+                quantity: parseFloat(coinage)
+                };
+
+        // history not sure if we want this in history here, probably in processMarket()....
+        // if (userData.history.length > 9)
+        //     userData.history.pop();
+        // userData.history.unshift('b' + ',' + amount.toFixed(2) + ',' + price.toFixed(2));
+
+        // statistics
+        //volume.bought += amount;
+    }
+    else if (command == 'sell'){
+        // check valid amount
+        if (amount <= 0.0) {
+            message.channel.send(response + ', that is an invalid amount.');
+            return;
+        }
+        // check user funds
+        if (userData.meatCoin < amount) {
+            message.channel.send(response + ', you do not have that much MeatCoin.');
+            return;
+        }
+        // check user funds
+        const value = amount;
+        if (userData.meatCoin < value) {
+            message.channel.send(response + ', you have insufficient meatCoin. ' + value + ' meatCoin is required.');
+            return;
+        }
+        //check that the user is not already buying
+        if (id in market[0]){
+            response += ', you can\'t create a sell posting, you are already buying. ya dingus!\n';
+            message.channel.send(response);
+            message.author.send('Your posting:\n' + id.padStart(18,'0') + '\tB ' + String(market[0][id].quantity).padStart(10,' ') + '\t@' + String(market[0][id].price).padStart(8,' ') + '\n');
+            return;
+        }
+        //check that the user is not also selling
+        if (id in market[1]){
+            response += ', you can\'t create a sell posting, you are already selling. Use !post adjust.\n';
+            message.channel.send(response);
+            message.author.send('Your posting:\n' + id.padStart(18,'0') + '\tS ' + String(market[1][id].quantity).padStart(10,' ') + '\t@' + String(market[1][id].price).padStart(8,' ') + '\n');
+            return;
+        }
+        response += ', is looking to unload some meat, get it while it\'s hot';
```

Comment:
Love this lmao
response += ', is looking to unload some meat, get it while it\'s hot';